iXBRLViewerPlugin/viewer/src/js/concept.js

```diff
@@ -36,7 +36,7 @@ Concept.prototype.referenceValuesAsString = function() {
 }
 
 Concept.prototype.references = function () {
-    if (!this._c.r) {
+    if (!this._c || !this._c.r) {
```

Comment:
Should `isTypedDimension` be similarly adjusted?  It looks like something similar would happen [here](https://github.com/Workiva/ixbrl-viewer/blob/f69a4f82edc68403305f59644708a15c7d15e23a/iXBRLViewerPlugin/viewer/src/js/search.js#L38) if a fact has a reference to an invalid concept.  Actually, that line looks like it has a bug and should be `isTypedDimension()`?

Reply:
Good point, and yes, there's a bug there which is why we weren't tripping over on undefined concepts.  Both now fixed.